plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/utils/ConfigurationUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka.utils;
+
+import com.google.common.collect.Maps;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public final class ConfigurationUtils
+{
+    private ConfigurationUtils() {}
+
+    public static Map<String, String> readProperties(List<File> resourcePaths)
```

Comment:
We could use Airlift's `ConfigurationLoader#loadPropertiesFrom` so I guess we don't need this method

Reply:
`Airlift's ConfigurationLoader#loadPropertiesFrom` doesn't seem to support yet to read multiple files. I can see it only supports reading from a single string path